### PR TITLE
Update Facebook marketing API version

### PR DESCRIPTION
**Pull request by thinhnd2104**
image

### CODE REVIEW

**Review by erictabalipa — APPROVED** [review]

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ class API:
     def __init__(self, account_id: str, access_token: str):
         self._account_id = account_id
         # design flaw in MyFacebookAdsApi requires such strange set of new default api instance
-        self.api = MyFacebookAdsApi.init(access_token=access_token, crash_log=False)
+        self.api = MyFacebookAdsApi.init(access_token=access_token, crash_log=False, api_version='v14.0')
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py:164]: You need to change the facebook sdk version in setup.py. Please sign the CLA also and bump the connector version in the dockerfile and make sure you allow maintainer edit your PR.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py (hunk 1 above)
Comment (thinhnd2104) [reply airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py:164]: SDK version was not changed. Still 13.0.0 but API version is 14.0 @marcosmarxm

**Review by sherifnada — CHANGES_REQUESTED** [review]
@thinhnd2104 can you please summarize the API version changes and whether any of them impact the existing functionality in this connector?

**Comment by thinhnd2104** [comment]
> 

With FB SDK: When API version changes but it only changes path on request uri. Ex: v13.0, v14.0. If not change API version, this connector does not work.
So my suggestion is we should add api version on Frontend and add default value is current version.

**Comment by marcosmarxm** [comment]
@thinhnd2104 can you sign the CLA too?

**Comment by marcosmarxm** [comment]
> With FB SDK: When API version changes but it only changes path on request uri. Ex: v13.0, v14.0. If not change API version, this connector does not work. So my suggestion is we should add api version on Frontend and add default value is current version.

@sherifnada what your opinion here?

**Comment by thinhnd2104** [comment]
> @thinhnd2104 exposing the version, while convenient on the surface, also potentially adds a lot of complexity because now the connector needs to support the features of all API versions. In any case, the first step to doing this is to understand how the changes in the new API version impact the connector's current functionality. My guess is it's much easier to upgrade this to the latest version than to maintain support for all possible API versions.

so how to get facebook data if API is not latest version?

**Comment by marcosmarxm** [comment]
> so how to get facebook data if API is not latest version?
wait facebook release the new sdk version, probably you can create an issue in fb github repo or create an issue in their support.

**Comment by Qaisarg** [comment]
@thinhnd2104 I am still facing the same issue is this issue resolved??

**Comment by thinhnd2104** [comment]
> @thinhnd2104 I am still facing the same issue is this issue resolved??
No :)) we need to wait new version from facebook.

**Comment by vladimir-remar** [comment]
@thinhnd2104 @marcosmarxm any chance to update to the new version https://pypi.org/project/facebook-business/

**Pull request closed without merging**